### PR TITLE
Fixes #22725 - Add newline to CA to prevent EOM error on registration.

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

File: templates/rhsm-katello-reconfigure.erb [hunk 1]
```diff
@@ -16,11 +16,13 @@ type -P subscription-manager >/dev/null || type -P subscription-manager-cli >/de
 # backup configuration during the first run
 test -f $CFG_BACKUP || cp $CFG $CFG_BACKUP
 
-#create the cert
+# create the cert and add newline to end of cert
 echo "$KATELLO_SERVER_CA_DATA" > $KATELLO_CERT_DIR/$KATELLO_SERVER_CA_CERT
+echo "" >> $KATELLO_CERT_DIR/$KATELLO_SERVER_CA_CERT
```
Comment (ekohl) [inline templates/rhsm-katello-reconfigure.erb]: You can also echo `"$KATELLO_SERVER_CA_DATA\n"` so you don't need a separate command.

File: templates/rhsm-katello-reconfigure.erb [hunk 2]
```diff
@@ -16,11 +16,13 @@ type -P subscription-manager >/dev/null || type -P subscription-manager-cli >/de
 # backup configuration during the first run
 test -f $CFG_BACKUP || cp $CFG $CFG_BACKUP
 
-#create the cert
+# create the cert and add newline to end of cert
```
Comment (ekohl) [inline templates/rhsm-katello-reconfigure.erb]: I'd say `ensure a newline`.

**Comment by chris1984** [comment]
@ekohl thanks and good to know for future PRS, I have updated the PR to reflect the changes and a better commit message.

**Comment by chris1984** [comment]
@ekohl updated.

**Pull request merged**